src/SonarQube.Client/Api/Requests/V5_20/GetQualityProfileChangeLogRequest.cs

```diff
@@ -53,6 +53,17 @@ public virtual string LanguageName
             set { /* not supported in this implementation */ }
         }
 
+        /// <summary>
+        /// This property does not exist before SonarQube 6.4 and is deliberately implemented to always return
+        /// the default value in order to prevent it from serialization.
+        /// </summary>
+        [JsonProperty("organization", DefaultValueHandling = DefaultValueHandling.Ignore), DefaultValue(null)]
```

Comment:
The new parameter has to be added to the interface (i.e. the interface is a superset of all the possible parameters for every version).
This is the v5_2 base class, which is the version in which the API was introduced.
The class for the later version of the API (6.5) inherits from this class and overrides the property so it can be set and serialized.